src/ui/public/styles/variables/bootstrap-mods.less

```diff
@@ -370,7 +370,7 @@
 @navbar-inverse-border:                 darken(@navbar-inverse-bg, 10%);
 
 // Inverted navbar links
-@navbar-inverse-link-color:             @gray-lighter;
+@navbar-inverse-link-color:             #ED9C06;
```

Comment:
Maybe you can define a constant here like they do with `@gray-lighter`. In fact, you have defined the same value for `@brand-info` in `colors.less` file.